products/resourcemanager/terraform.yaml

```diff
@@ -37,6 +37,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       decoder: templates/terraform/decoders/avoid_meaningless_project_update.erb
   Folder: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+    # iam_policy: !ruby/object:Api::Resource::IamPolicy
```

Comment:
Hm, I don't see what commenting this out does to the downstream diffs - wondering, can you revert this change?

Reply:
I got the autogen working now, but it has incorrect logic. Folders and Organizations IAM can't use custom project roles but the doc has that in there.